### ISSUE TITLE
As a developer, I should fix some issues found by user testing in the Workspace

### ISSUE THREAD

**Issue by jblom** [beeldengeluid/labo-components#168]
@lilimelgar commented on [Mon Jun 11 2018](https://github.com/CLARIAH/wp5_mediasuite/issues/436)

These are the fixes from the fourth iteration (in order of importance):

Important:
- [ ] Translator-like function for labels in the workspace, most important: Classification = Codes, and Metadata = Metadata card, Segments = Fragments. Ideally these names should be changed also in the code (annotation tool), but temporary solution is also OK.
- [ ] In the bookmark view: check why ordering function does not work sometimes (e.g., user project "Test 11": ordering per title) --added as new issue: #515 
- [ ] In the Bookmark view, add links to the Codes, so they bring the user to the Code view
- [ ]  Change display name of datasets (from Nisv-catalog-aggr to Ckan collection title)
- [ ] For all date fields, add a tool-tip (mouse-over function) indicating which date field was used (see how this is done in the resource viewer for example)
- [ ] Show Item ID in the bookmark list (it can be in the details)
- [x] In the code view, add to search: code, title and date of bookmarked item, and any other words in the user annotations (does not seem to work well when searching in all collections)
- [x] In the bookmark view, add floating selector for bulk actions, so users don't have to scroll down to pick up an option (delete, export)
- [x] In the Saved Queries tab, include option to "Add new query" (this can be added when the Comparative search tool is ready? to reuse that button/function?)
- [x] Add pop-up info when user deletes bookmarks informing that it takes time
- [ ] In the links view, test the Search functionality further, it does not work with all projects, but it works well with the Bookmark test project
- [x] Add a new filter or icon in the bookmark view: viewable/non-viewable (this depends on issue #352 being solved)
- [ ] In the links view, consider hiding the full URL behind the label (it now displays very long URLs that may not be necessary for the user to see in display)
- [ ] If possible, add button for clearing-up all filters (mainly in the Bookmark view where there are more)

---

@lilimelgar commented on [Mon Jul 16 2018](https://github.com/CLARIAH/wp5_mediasuite/issues/436#issuecomment-405195112)

The fixes/improvements listed above are still valid (required) after Summer school test.

---

@lilimelgar commented on [Mon Jul 16 2018](https://github.com/CLARIAH/wp5_mediasuite/issues/436#issuecomment-405273594)

@jblom after testing in summer school, the fixes above are still needed. However, these can wait until we evaluate the next steps/iteration for the Workspace.
